vpr/src/device/rr_graph_builder.h

```diff
@@ -55,6 +55,9 @@ class RRGraphBuilder {
     /** @brief Clear all the underlying data storage */
     void clear();
 
+    /** @brief Set the node coordinate */
```

Comment:
This probably isn't too cpu critical so can leave here.  But should get a QoR spreadsheet to check we haven't slowed things down noticably; if there's a slowdown, marking this function as inline and adding the function body to the header should recapture the speed.  For very short functions like this, inline works well.

Reply:
I have shared the QoR sheets. Please let me know if this needs to be inlined after seenig QoR comparisons.